pkg/sql/plan/build_load.go

```diff
@@ -86,6 +86,9 @@ func buildLoad(stmt *tree.Load, ctx CompilerContext) (*Plan, error) {
 	tableDef.Createsql = string(json_byte)
 	node1.TableDef = tableDef
 	node1.ObjRef = objRef
+	if stmt.Param.Tail.FromJsonLine {
```

Comment:
if you Marshal stmt.Param to Createsql.
Why you set this to ExtraOptions？ may be it's not necessary, and ExtraOptions also is not a good choice

Reply:
Ok，I get it. I will remove this. 
Thx.